homeassistant/components/tts/strings.json

```diff
@@ -0,0 +1,60 @@
+{
+  "services": {
+    "say": {
+      "name": "Say a TTS message",
+      "description": "Says something using text-to-speech on a media player.",
+      "fields": {
+        "entity_id": {
+          "name": "Entity",
+          "description": "Media players to play the message."
+        },
+        "message": {
+          "name": "Message",
+          "description": "The text you want to convert into speech so that you can listen to it on your device."
+        },
+        "cache": {
+          "name": "Cache",
+          "description": "Stores this message locally so that when the text is requested again, the output can be  produced more quickly."
```

Comment:
```suggestion
          "description": "Stores this message locally so that when the text is requested again, the output can be produced more quickly."
```